core/transact.go

```diff
@@ -264,6 +272,7 @@ func waitForTxInBlock(ctx context.Context, c *protocol.Chain, tx *bc.Tx, height
 type submitArg struct {
 	Transactions []*txbuilder.Template
 	wait         chainjson.Duration
+	WaitFor      string `json:"wait_for"` // values none, confirmed, processed. default: processed
```

Comment:
nit: if we use the values `confirmed` and `processed` maybe this should be `wait_until`?